### PR TITLE
Bug/footer to bottom

### PR DESCRIPTION
**Pull request by rossneilson**
## Description

Currently, the footer is floating and not sticking to the bottom of the screen.
Fix involved changing position to absolute and bottom to 0, making it always render at the bottom of the page.

It also seems there is a minHeight property set to 200% which leaves a huge empty space if the page doesn't have much content. If this is intentional I can remove the change.

## How Has This Been Tested?

Running locally using local subgraph 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/views/Sales/index.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export function SalesView() {
 
   return (
     <SaleContext.Provider value={{ SaleShow, setSaleShow }}>
-      <AbsoluteContainer minHeight="200%" inner={false} noPadding={true}>
```
Comment (rossneilson) [inline src/views/Sales/index.tsx:93]: If this is something intentional please let me know and I will remove it.

Comment (adamazad) [reply src/views/Sales/index.tsx:93]: This was introduced by an old developer. It is indeed an extra component in the tree.

**Comment by adamazad** [comment]
Is it possible to use Flexbox to patch this?

**Comment by rossneilson** [comment]
> Is it possible to use Flexbox to patch this?

Technically it is possible but it is a bit overkill for a basic layout. 
The parent could be made into a flexbox container with direction column and the header, content and footer spaced around it. However, this would be more likely to break other parts of the UI without any benefit. 
Is there a specific reason to use flexbox?

File: src/views/Sales/index.tsx (hunk 1 above)
Comment (rossneilson) [reply src/views/Sales/index.tsx:93]: I'll keep the component for now since it has some stylings in it. However I'll keep the minHeight removed.

**Comment by adamazad** [comment]
> > Is it possible to use Flexbox to patch this?
> 
> Technically it is possible but it is a bit overkill for a basic layout.
> The parent could be made into a flexbox container with direction column and the header, content and footer spaced around it. However, this would be more likely to break other parts of the UI without any benefit.
> Is there a specific reason to use flexbox?

Because I did fix this issue with flexbox, but if this works, then we can merge it :)

**Review by adamazad — APPROVED** [review]

**Pull request merged**